core/html_api.php

```diff
@@ -642,7 +642,8 @@ function print_manage_menu( $p_page = '' ) {
 
 	# Plugins menu items - these are cooked links
 	foreach( $t_menu_options as $t_menu_item ) {
-		echo '<li>', $t_menu_item, '</li>';
+		$t_active = $p_page && strpos( $t_menu_item, $p_page ) !== false ? 'class="active"' : '';
```

Comment:
For the comment above, I don't know what is meant by cooked links.  Can you provide an example of a link to make it more readable?